app/Http/Controllers/AdminDash.php

```diff
@@ -323,73 +270,25 @@ public function updateController(Request $request, $id) {
             } elseif ($request->input('api_exempt') == 1) {
                 $user->api_exempt = 1;
             }
-            $user->twr_solo_fields = $request->input('twr_solo_fields');
-            $user->twr_solo_expires = $request->input('twr_solo_expires');
-            
+
             $user->status = $request->input('status');
             $user->visitor_from = $request->input('visitor_from');
             $user->save();
 
-            if ($user->hasRole(['atm', 'datm', 'ta', 'ata', 'wm', 'awm', 'fe', 'afe', 'ec']) == true) {
```

Comment:
Out of scope for this PR, but this bit of logic hurts me. Is there no other way to update what roles a user has?

Reply:
This statement is simply saying: `If the user has one of these staff roles, then do x`

Reply:
Your reference is to code that I replaced. What logic in particular are you referring to here and I'll take a look at it?

Reply:
Done